crates/nu-parser/src/parser.rs

```diff
@@ -450,25 +450,22 @@ fn parse_short_flags(
 ) -> Option<Vec<Flag>> {
     let arg_span = spans[*spans_idx];
 
-    let arg_contents = working_set.get_span_contents(arg_span);
+    let arg_contents = working_set.get_span_contents(arg_span).to_vec();
```

Comment:
We don't need to materialize in this case.

Reply:
Yep, we can remove this if we add the `break`.